build-system/amp.extern.js

```diff
@@ -299,6 +299,7 @@ window.vg;
  * @type {function(*)}
  */
 let ReactRender = function() {};
+let RRule;
```

Comment:
Does this do anything? there's no type info 😛

Reply:
It's used by the lines below :)